src/dbnode/server/server.go

```diff
@@ -1511,3 +1519,13 @@ func (t *topoMapProvider) TopologyMap() (topology.Map, error) {
 
 	return t.t.Get(), nil
 }
+
+func emitBootstrappersGauge(scope tally.Scope, bs []string) tally.Gauge {
```

Comment:
Yeah as @robskillington was saying we'll need to emit this regularly. You may want to just create a simple type thats something like:

```golang
type bootstrapGaugeEmitter struct {
    sync.Mutex
    running bool
    doneCh chan struct{}
    bootstrappers []string
}

func (b *bootstrapGaugeEmitter) Start() {
    b.Lock()
    defer b.Unlock()
    if b.running {
        return
    }

    b.doneCh = make(chan struct{})
    b.running = true
    go func() {
        for {
            select {
            case <-b.doneCh:
                return
            default:
                 // All the logic for emitting the gauge. Will need to acquire lock to get most up to date bootstrappers.
            }
        }
    }()
}

func (b *bootstrapGaugeEmitter) Stop() {
    b.Lock()
    defer b.Unlock()
    if !b.running {
        return
    }
    close(b.doneCh)
}

func (b *bootstrapGaugeEmitter) Update(bootstrappers []string) {
    b.Lock()
    defer b.Unlock()
    b.bootstrappers = bootstrappers
}
```

I hacked that up in the browser so there may be some mistakes but you get the idea

Reply:
Awesome, thanks for the pointers! But why the lock, can't we just use a different channel where we pass the bootstrappers to?

Reply:
Sure you could implement this with a channel too, I just work with locks a lot so maybe that felt simpler to me. Whatever works.

Reply:
So I tried it with a second channel first but ultimately went with your suggestion (and built upon it) because I liked it better. I also moved the type to `x/instrument/metrics.go` but I'm not 100% certain about the location & naming. Do let me know if I should move it somewhere else and/or rename the file or such.